enterprise_learner_portal/api/v1/serializers.py

```diff
@@ -0,0 +1,74 @@
+
+
+from rest_framework import serializers
+
+try:
+    from lms.djangoapps.certificates.api import get_certificate_for_user
+    from lms.djangoapps.program_enrollments.api.api import (
+        get_due_dates,
+        get_course_run_url,
+        get_emails_enabled,
+        get_course_run_status
+    )
+except ImportError:
+    get_certificate_for_user = None
+    get_due_dates = None
+    get_course_run_url = None
+    get_emails_enabled = None
+    get_course_run_status = None
+
+
+class EnterpriseCourseEnrollmentSerializer(serializers.Serializer):
+    """
+    A serializer for course enrollment information for a given course
+    and enterprise customer user.
+    """
+
+    def to_representation(self, instance):
+        representation = super(EnterpriseCourseEnrollmentSerializer, self).to_representation(instance)
+
+        request = self.context['request']
+        user = self.context['user']
+
+
+
+# course_run_status
```

Comment:
need to write the logic for this still (porting it over from program_enrollment code in the platform)